src/components/tree/TreeMenu.tsx

```diff
@@ -0,0 +1,101 @@
+import {FC} from "react";
+import {Section} from "../../types/treeType";
+import {Props} from "../../types/treeMenuTypes";
+
+import "./TreeMenu.css";
+
+
+const TreeMenu: FC<Props> = ({info, treeSections, show, toggleShow}) => {
+  return (
+    <>
+      <div className="tree-menu">
+        <button
+          className={`btn btn-root root ${
+            !show || show === "root" ? "active" : "inactive"
+          }`}
+          title="Show only root"
+          onClick={() =>
+            toggleShow("root" as Section)
+          }
+          key="menu-root"
+        >
+          <strong>{(info.root || { title: "" }).title}</strong>
+          <small>{treeSections.root.length} articles</small>
+        </button>
+
+        <button
+          className={`btn btn-trunk trunk ${
+            !show || show === "trunk" ? "active" : "inactive"
+          }`}
+          title="Show only trunk"
+          onClick={() =>
+            toggleShow("trunk" as Section)
+          }
+          key="menu-trunk"
+        >
+          <strong>{(info.trunk || { title: "" }).title}</strong>
+          <small>{treeSections.trunk.length} articles</small>
+        </button>
+
+        {treeSections.branch_1 && (
+          <div className="btn-branches" key="section-branches">
+            <button
+              key="menu-branch_1"
+              className={`btn btn-branch branch ${
+                !show || show === "branch_1" ? "active" : "inactive"
+              }`}
+              title="Show only branch 1"
+              onClick={() =>
+                toggleShow("branch_1" as Section)
+              }
+            >
+              <strong>{(info.branch_1 || { title: "" }).title}</strong>
```

Comment:
Hi @SantiSite first time we share, just want to suggest something you can do in cases like this
```suggestion
              <strong>{info.branch_1?.title ?? ""}</strong>
```

You can use [optional chaning](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) and also the [nulish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)

Reply:
Hi @dani-vh thank you for you suggestions, I will take them into account. :+1: